api/specs/web-server/_projects_crud.py

```diff
@@ -50,6 +50,8 @@
 async def create_project(
     _params: Annotated[ProjectCreateParams, Depends()],
     _create: ProjectCreateNew | ProjectCopyOverride,
+    x_simcore_parent_project_uuid: str | None = Header(default=None),
+    x_simcore_parent_node_id: str | None = Header(default=None),
```

Comment:
I would maybe suggest to make this `x_simcore_parent_node_uuid` instead and to also use `ProjectID` and `NodeID` models here.

Reply:
sorry I thought I answered already that one.
the node ID is called node ID most of the time and it is not really "unique" it is only unique in conjunction with a project UUID.

Reply:
Makes sense, thanks for the clarification